### ISSUE TITLE
additional files in downloaded StackBlitz zip file can cause confusion

### ISSUE THREAD

**Issue by mMerlin** [angular/angular#32524]
# 📚 Docs or angular.io bug report

### Description

The `Building locally` section says (abbreviated) to download the project code from StackBlitz, and use it to replace the `/src` folder of an empty local project.  The downloaded zip file includes files `angular.json`, `package.json` and `tsconfig.json` outside of the source folder.  The deployment information should mention them, and what to do with them.  In this case they are ignored and thrown away, but I suspect that information from them will be needed if extra packages have been installed.  Since the zip file contains more than the `/src` folder, need to be a bit more careful just where it is unzipped.  My first attempt unzipped to the root folder for the local angular project, then `ng new …` failed because it saw the json files, and refused to create a new project in what it saw as an existing project folder.  Including information about the files, and a brief description of what they are used for would avoid confusion.

## 🔬 Minimal Reproduction

Download the start application from StackBlitz.  Unzip.  See the actual extracted contents, compared to what the deployment page describes.

### What's the affected URL?**

https://angular.io/start/deployment

`Building locally` second paragraph, mixed with creating a new Angular CLI workspace, and replacing the `/src` folder.

**Comment by kapunahelewong** [comment]
Hi @mMerlin, thanks for explaining your concerns. I'm going through the Getting Started and making some updates to help clarify things like this (PR on is in the works but not submitted yet) and we're currently working on the StackBlitz download part (where you copy your `src` directory). I'll reference this issue once my PR is in. Feel free to log any more thoughts here and tag me so I can field them.

**Comment by Orodan** [comment]
Hi @kapunahelewong, did you have time to work on this or do you need any help :) ? I can take care of clarifying the deployment sections regarding the stackblitz files if needed.

**Comment by kapunahelewong** [comment]
Ah, thank you very much @Orodan. 😊I think I'm ok, as I have some new steps I haven't pushed up yet that I am going to test locally tomorrow. Maybe if I run into a snag I can tag you? 🙈🙂

**Comment by Orodan** [comment]
@kapunahelewong sure, I'll be happy to help :)

**Comment by rockument69** [comment]
@kapunahelewong : Please see [my comment](https://github.com/angular/angular/issues/32527#issuecomment-634324507) in [32527](https://github.com/angular/angular/issues/32527), as this could just be one of the steps, and we could then close this issue (as well as that one).

**Comment by aikidave** [comment]
@mgechev If we can get the default project on Stackblitz to use strict mode, we can update these instructions so they're much easier for the user to understand. Can you help with this?

**Comment by IgorMinar** [comment]
I went over the tutorial and was able to follow the instructions and successfully build the app with CLI @ v12.

I agree that it might be a bit surprising to ignore the config files at the root, but these files are not necessary and in fact should be ignored.

Rather than making the instructions more complicated, I'd recommend that we use the new [stackblitz webcontainers](https://blog.stackblitz.com/posts/introducing-webcontainers/) in this tutorial and run the CLI from stackblitz. This change would make most of the text on the "deployment" page unnecessary. I filed #42404 to track this feature request.
